services/question/src/controllers/questionController.ts

```diff
@@ -1,11 +1,243 @@
 import { Request, Response } from 'express';
+import { handleError, handleNotFound, handleBadRequest, handleSuccess } from '../utils/helpers';
 import { Question } from '../models/questionModel';
 
+/**
+ * This endpoint allows the retrieval of all the questions in the database (or can filter by optional parameters).
+ * @param req
+ * @param res
+ */
 export const getQuestions = async (req: Request, res: Response) => {
-    const questions = await Question.find();
-    const questionTitles = questions.map(q => q.title);
-    res.status(200).json({
-        message: 'These are all the question titles:' + questionTitles,
-    });
-    return;
+    try {
+        const { title, description, topics, difficulty } = req.query;
+        // eslint-disable-next-line @typescript-eslint/no-explicit-any
```

Comment:
I think we can just globally disable this. What do you think?

Reply:
Yes, I think so too!